### PR TITLE
Fix API_PREFERRED_VERSIONS.autoscaling to horizontalpodautoscalers

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Mistake naming this version

@spadgett

### CODE REVIEW

File: dist/origin-web-common.js [hunk 1]
```diff
@@ -1425,11 +1425,11 @@ angular.module('openshiftCommonUI')
 
 angular.module('openshiftCommonServices')
   .constant('API_PREFERRED_VERSIONS', {
-      autoscaling:                  {group: 'autoscaling',            resource: 'horizontalpodautoscalers' },
+      horizontalpodautoscalers:     {group: 'autoscaling',            resource: 'horizontalpodautoscalers' },
```
Comment (spadgett) [inline dist/origin-web-common.js]: nit: alpha

Comment (benjaminapetersen) [reply dist/origin-web-common.js]: lol not `alpha` as in api, `alpha` as in alphabetize.  I will fix :)

**Comment by benjaminapetersen** [comment]
Updated

**Comment by benjaminapetersen** [comment]
Sneaking in `'deploymentconfigs/instantiate'` as well.

**Comment by spadgett** [comment]
> Sneaking in 'deploymentconfigs/instantiate' as well.

Might want to add `deploymentconfigs/rollback` in that case, too.

**Comment by benjaminapetersen** [comment]
will do.

**Comment by benjaminapetersen** [comment]
@spadgett thoughts on this?  Could get 2-3 of the larger PRs merged in today then.

**Pull request merged**